Scripts/Setup/setupExperiment.sh

```diff
@@ -404,8 +407,12 @@ if [[ "${DATATYPE}" == 'CESM' ]] || [[ "${DATATYPE}" == 'CCSM' ]] || [[ "${DATAT
 elif [[ "${DATATYPE}" == 'CFSR' ]]; then
   ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE_PLEV}" 'Vtable.CFSR_plev'
   ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE_SRFC}" 'Vtable.CFSR_srfc'
-elif [[ "${DATATYPE}" == 'ERA-I' ]]; then
-  ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE}" 'Vtable'
+else
+  if [[ -n "${VTABLE}" ]]; then 
+    ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE}" 'Vtable'
+  else 
+    echo "VTABLE variable is needed but not defined. Aborting." 
```

Comment:
does this actually abort?
btw. 'elif' would have worked, too

Reply:
Fixed. Done.